woocommerce/woocommerce
# Replace "Personalize Your Store" Task with "Choose Your Theme"

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Replaces the "Personalize Your Store" Task with a simple "Choose Your Theme" task that redirects to the WordPress Block Themes selection screen.

- The original task ID (`appearance`) is retained to ensure Woo Express customizations are not impacted.
- The task is marked as complete when the themes screen is viewed.
- This is a port of the "Choose Your Theme" task on Woo Express.
- This is a stopgap measure until the Customize Your Theme Task is deployed in Core.

https://github.com/woocommerce/woocommerce/assets/9312929/6fbf0380-5854-4edb-9d8b-91c4587401bc


Closes https://github.com/woocommerce/woocommerce/issues/40238 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure that Twenty Twenty-Three is the current theme
2. View the WooCommerce home screen.
4. Observe the "Choose your theme" task is present and not completed
5. Observe the "Personalize your store" task is not present
6. Click the "Choose your theme" task
7. Observe the redirect to Appearance > Themes
8. Install and activate any theme
9. View the WooCommerce home screen.
10. Observe the "Choose your theme" task is completed



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
